livecheck/utils/portage_test.py

```diff
@@ -1,12 +1,11 @@
-"""Configuration for Pytest."""
```

Comment:
Rename this file to `tests/utils/test_portage.py`. See how `tests` at the root is used:

- https://github.com/Tatsh/baldwin/tree/master/tests
- https://github.com/eadwinCode/django-ninja-extra/tree/master/tests

Do not create `__init__.py` in these dirs as it is not required.